### PR TITLE
balloon_check: close session in the end of test

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: qemu/tests/balloon_check.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def run_balloon_sub_test(self, test, params, env, test_tag):
             qemu_quit_after_test = 1
         if params.get("session_need_update", "no") == "yes":
             timeout = int(self.params.get("login_timeout", 360))
```
Comment (huangyum) [inline qemu/tests/balloon_check.py]: Is this line useless?

Comment (xutian) [inline qemu/tests/balloon_check.py]: right, thanks your comment

**Comment by xutian** [comment]
@huangyum updated, please review again. thanks

**Comment by huangyum** [comment]
@xutian , seems you forgot to push the update.

**Comment by xutian** [comment]
done, please check

**Comment by huangyum** [comment]
(1/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict.balloon-migrate: PASS (247.96 s)
 (2/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict.balloon-reboot: PASS (368.01 s)
 (3/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict.balloon-shutdown: PASS (186.27 s)
 (4/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict.balloon-stop_continue: PASS (191.76 s)
 (5/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_enlarge.balloon-migrate: PASS (241.32 s)
 (6/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_enlarge.balloon-reboot: PASS (375.59 s)
 (7/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_enlarge.balloon-shutdown: PASS (192.84 s)
 (8/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_enlarge.balloon-stop_continue: PASS (224.09 s)
 (9/9) smp_1.3226m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict_and_enlarge.balloon_base: PASS (194.98 s)

LGTM
Acked-by: Yumei Huang yuhuang@redhat.com

**Comment by xutian** [comment]
merged, and thanks @huangyum test results

**Pull request merged**